src/PanelTraits/Breadcrumb.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Breadcrumb
+{
+    /*
+    |--------------------------------------------------------------------------
+    |                              BREADCRUMB
+    |--------------------------------------------------------------------------
+    */
+
+    protected $items;
+
+    /**
+     * Parse the word.
+     *
+     * @return string the parsed word.
+     **/
+    protected function parseWord($word)
+    {
+        $words = explode('_', $word);
+        $string = '';
+        foreach ($words as $key => $word) {
+            $string .= $key > 0 ? ' ' . ucfirst($word) : ucfirst($word);
+        }
+        return $string;
+    }
+
+
+    /**
+     * Generate the actual breadcrumb.
```

Comment:
"actual" is redundant - unless there's a "Generate the imaginary breadcrumb" function :P